packages/phishing-controller/src/PhishingController.test.ts

```diff
@@ -652,14 +724,21 @@ describe('PhishingController', () => {
     nock(PHISHING_CONFIG_BASE_URL)
       .get(METAMASK_STALELIST_FILE)
       .reply(200, {
-        blocklist: [],
-        fuzzylist: [],
-        tolerance: 0,
-        allowlist: [],
-        version: 0,
-        lastUpdated: 0,
+        data: {
+          eth_phishing_detect_config: {
+            allowlist: [],
+            blocklist: ['xn--myetherallet-4k5fwn.com'],
```

Comment:
Possibly a copy+paste mistake? Looks like the old version of this test was returning a blank response for all stalelist properties, since it's meant to test the error handling behavior of the hotlist failing to update.
```suggestion
            blocklist: [],
```

Reply:
Fixed.